backend/cmmgen.ml

```diff
@@ -816,8 +817,15 @@ and transl_ccall env prim args dbg =
     | Untagged_int -> (typ_int, (fun i -> tag_int i dbg))
   in
   let typ_args, args = transl_args prim.prim_native_repr_args args in
-  let op = cextcall prim args dbg typ_res typ_args in
-  wrap_result op
+  wrap_result
+    (Cop(Cextcall { func = Primitive.native_name prim;
+                    builtin = false;
+                    returns = true;
+                    effects = Arbitrary_effects;
+                    coeffects = Has_coeffects;
+                    ty = typ_res; alloc = prim.prim_alloc;
+                    ty_args = typ_args },
+     args, dbg))
```

Comment:
This change is not related (it probably snack in from a previous issue with rebase, fixed in https://github.com/ocaml-flambda/flambda-backend/pull/79).
Please revert it to invoke `cextcall` as before and pass `true` for the new `returns` argument.